agent/src/index.ts

```diff
@@ -203,12 +137,23 @@ function initializeDatabase(dataDir: string) {
 
 export async function initializeClients(
     character: Character,
-    runtime: IAgentRuntime
+    runtime: AgentRuntime,
+    dc: any
 ) {
     const clients = [];
     const clientTypes =
         character.clients?.map((str) => str.toLowerCase()) || [];
 
+    if (clientTypes.includes("direct")) {
+        dc.registerAgent(runtime);
```

Comment:
I think we want to always register these, direct is poorly named but provides a rest API for managing agents

Reply:
I wanted to split it off because then the user would have control over which agent the "direct" client is talking to, instead of defaulting to characters[0]